de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java

```diff
@@ -0,0 +1,205 @@
+
+  protected FSKDBViewNodeModel() {
+    super(IN_OUT_TYPES, IN_OUT_TYPES, VIEW_NAME);
+  }
+
+  /**
+   * The logger is used to print info/warning/error messages to the KNIME console and to the KNIME
+   * log file. Retrieve it via 'NodeLogger.getLogger' providing the class of this node model.
+   */
+  private static final NodeLogger LOGGER = NodeLogger.getLogger(FSKDBViewNodeModel.class);
+
+  /**
+   * The settings key to retrieve and store settings shared between node dialog and node model. In
+   * this case, the key for the online repository URL String that should be entered by the user in
+   * the dialog.
+   */
+  private static final String KEY_REPOSITORY_LOCATION = "repository_location";
+
+  /**
+   * The default online repository URL String.
+   */
+  private static final String DEFAULT_REPOSITORY_LOCATION = "https://knime.bfr.berlin/backend/";
+
+  /**
+   * The settings model to manage the shared settings. This model will hold the value entered by the
+   * user in the dialog and will update once the user changes the value. Furthermore, it provides
+   * methods to easily load and save the value to and from the shared settings (see: <br>
+   * {@link #loadValidatedSettingsFrom(NodeSettingsRO)}, {@link #saveSettingsTo(NodeSettingsWO)}).
+   * <br>
+   * Here, we use a SettingsModelString as the online repository URL is a String. Also have a look
+   * at the comments in the constructor of the {@link FSKDBViewNodeDialog} as the settings models
+   * are also used to create simple dialogs.
+   */
+  private final SettingsModelString m_repositoryLocationSettings =
+      createRepositoryLocationSettingsModel();
+
+
+  /**
+   * A convenience method to create a new settings model used for the online repository URL String.
+   * This method will also be used in the {@link FSKDBViewNodeDialog}. The settings model will sync
+   * via the above defined key.
+   * 
+   * @return a new SettingsModelString with the key for the online repository URL String
+   */
+  static SettingsModelString createRepositoryLocationSettingsModel() {
+    return new SettingsModelString(KEY_REPOSITORY_LOCATION, DEFAULT_REPOSITORY_LOCATION);
+  }
+
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected void saveSettingsTo(final NodeSettingsWO settings) {
+    m_repositoryLocationSettings.saveSettingsTo(settings);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected void loadValidatedSettingsFrom(final NodeSettingsRO settings)
+      throws InvalidSettingsException {
+    m_repositoryLocationSettings.loadSettingsFrom(settings);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected void validateSettings(final NodeSettingsRO settings) throws InvalidSettingsException {
+    m_repositoryLocationSettings.validateSettings(settings);
+  }
+
+  @Override
+  public FSKDBViewRepresentation createEmptyViewRepresentation() {
+    FSKDBViewRepresentation representation = new FSKDBViewRepresentation();
```

Comment:
The view representation is not initializing its tableID. Is it intended?

Reply:
Yes in the next pull request